src/mtev_defines.h

```diff
@@ -226,6 +228,7 @@ static inline int portable_readdir_r(DIR *dirp, struct dirent *entry, struct dir
   return readdir_r(dirp, entry, result);
 #pragma GCC diagnostic pop
 }
```

Comment:
```
#else 
#define portable_readdir_r readdir_r
```

Reply:
I see, so we duplicate the `portable_readdir_r`, but seems unavoidable.